File: 1828.c
#include <stdio.h>
#include <string.h>

int sheldonWins(char *a, char *b) {
    return (strcmp(a, "tesoura") == 0 && strcmp(b, "papel") == 0) ||
           (strcmp(a, "papel") == 0 && strcmp(b, "pedra") == 0) ||
           (strcmp(a, "pedra") == 0 && strcmp(b, "lagarto") == 0) ||
           (strcmp(a, "lagarto") == 0 && strcmp(b, "Spock") == 0) ||
           (strcmp(a, "Spock") == 0 && strcmp(b, "tesoura") == 0) ||
           (strcmp(a, "tesoura") == 0 && strcmp(b, "lagarto") == 0) ||
           (strcmp(a, "lagarto") == 0 && strcmp(b, "papel") == 0) ||
           (strcmp(a, "papel") == 0 && strcmp(b, "Spock") == 0) ||
           (strcmp(a, "Spock") == 0 && strcmp(b, "pedra") == 0) ||
           (strcmp(a, "pedra") == 0 && strcmp(b, "tesoura") == 0);
}

int main() {
    int n, i;
    char option1[20], option2[20];

    scanf("%d", &n);
    for (i = 0; i < n; i++) {
        scanf("%s %s", option1, option2);

        if (strcmp(option1, option2) == 0) {
            printf("Caso #%d: De novo!\n", i + 1);
        } else if (sheldonWins(option1, option2)) {
            printf("Caso #%d: Bazinga!\n", i + 1);
        } else {
            printf("Caso #%d: Raj trapaceou!\n", i + 1);
        }
    }

    return 0;
}

File: 1828.py
def sheldon_wins(a, b):
    return ((a == "tesoura" and b == "papel") or
            (a == "papel" and b == "pedra") or
            (a == "pedra" and b == "lagarto") or
            (a == "lagarto" and b == "Spock") or
            (a == "Spock" and b == "tesoura") or
            (a == "tesoura" and b == "lagarto") or
            (a == "lagarto" and b == "papel") or
            (a == "papel" and b == "Spock") or
            (a == "Spock" and b == "pedra") or
            (a == "pedra" and b == "tesoura"))

n = int(input())
for i in range(1, n + 1):
    opcao1, opcao2 = input().split()
    if opcao1 == opcao2:
        print(f"Caso #{i}: De novo!")
    elif sheldon_wins(opcao1, opcao2):
        print(f"Caso #{i}: Bazinga!")
    else:
        print(f"Caso #{i}: Raj trapaceou!")